### ISSUE TITLE
Remove early documentation from the repo 

### ISSUE THREAD

**Issue by HadrienGardeur** [readium/ts-toolkit#36]
While this was initially meant to be a monorepo hosting the Readium Web project, things shifted to a different approach a while back.

The Readium Web project is now split in two between the TypeScript and Go toolkits but this repo still contains documentation of what it was before in its early days.

We need to remove these documents from the repo and once that's done, I'm also in favor of merging back the `dev` branch into `main`. We're sending the wrong signal when people check this repo and see that the last commit was several years ago.

**Comment by qnga** [comment]
> We need to remove these documents from the repo and once that's done, I'm also in favor of merging back the dev branch into main. We're sending the wrong signal when people check this repo and see that the last commit was several years ago.

On mobile we changed the default branch to `develop`. IMHO, the default branch should be the one contributions are expected to be based on.

**Comment by mickael-menu** [comment]
Mainly because the default branch is the one used by default when creating a PR or cloning the repo. It's confusing for contributors when it's `main`.

As long as the TS toolkit doesn't have releases and stable versions, I would keep a single branch and work there.

**Comment by HadrienGardeur** [comment]
Yeah, I agree that until we've reached a 1.0 milestone, a single branch would work best.

I also noticed an inconsistency in how we name those branches:

- `develop` for Readium Mobile
- `dev` for Readium Web

**Comment by chocolatkey** [comment]
I'll go ahead and adapt to the readium mobile naming, since there's less people dependent on the TS toolkit
Edit: Done, dev --> develop and develop is default

**Comment by chocolatkey** [comment]
I noticed the docs are published to https://readium.org/ts-toolkit/ (via GitHub Pages). If we remove the docs completely, that will disappear. Do we want to do something about that?

**Comment by HadrienGardeur** [comment]
That's fine, these docs are not up to date and we'll cover things through https://readium.org/web moving forward.

We'll also need a good README and docs for both ts-toolkit and go-toolkit but that's a separate discussion.
